data/xml/2024.jeptalnrecital.xml

```diff
@@ -0,0 +1,1765 @@
+      <author><first>Erwan</first><last>Pépiot</last></author>
+      <pages>589–597</pages>
+      <abstract>Cette étude porte sur les productions d’enfants francophones francilien•nes âgé•es de 8 à 10 ans, en lecture et en parole semi-spontanée. Deux groupes ont été enregistrés : des élèves d’une école privée favorisée (10 filles et 8 garçons), et des enfants scolarisés dans une école publique défavorisée (8 filles et 10 garçons). La F0 moyenne et la modulation de F0 ont été analysées. Les filles présentent une F0 moyenne significativement plus élevée que les garçons dans les deux écoles. La différence étant légèrement plus marquée chez les enfants de l’école favorisée. Aucune corrélation significative n’a été trouvée entre la taille des locuteur•rices et leur F0 moyenne. La modulation de F0 est significativement plus élevée chez les filles au sein de l’école favorisée, mais très similaire entre les deux genres dans l’école défavorisée. Indépendamment du genre, la modulation de F0 est plus forte chez les enfants issus de milieu favorisé.</abstract>
+      <url hash="cbf11294">2024.jeptalnrecital-jep.61</url>
+      <language>fra</language>
+      <bibkey>pepiot-2024-voix</bibkey>
+    </paper>
+    <paper id="62">
+      <title>i<fixed-case>H</fixed-case>ist et i<fixed-case>S</fixed-case>catter, outils en ligne d’exploration interactive de données : application aux valeurs aberrantes de f0 et de formants</title>
+      <author><first>Nicolas</first><last>Audibert</last></author>
+      <pages>598–607</pages>
+      <abstract>Les mesures aberrantes d’un point de vue statistique (outliers) doivent être traitées avec précaution, ce qui peut être compliqué en pratique lorsque la quantité de données devient importante. Afin de faciliter l’inspection des valeurs situées à la marge des distributions, nous proposons deux outils développés avec R/Shiny, disponibles sous forme d’applications en ligne utilisables par des non-spécialistes et distribués gratuitement sous licence GPL. Ces applications permettent de paramétrer la visualisation et d’explorer de façon interactive des distributions via des histogrammes, et les relations entre variables quantitatives via des nuages de points. Deux cas d’utilisation appliqués à des données de parole sont présentés pour illustrer les principales fonctionnalités de ces outils, à partir de mesures acoustiques extraites par Praat : l’ajustement des valeurs limites pour la détection automatique de la fréquence fondamentale, et l’identification de valeurs erronées de formants.</abstract>
+      <url hash="39fd8f83">2024.jeptalnrecital-jep.62</url>
+      <language>fra</language>
+      <bibkey>audibert-2024-ihist</bibkey>
+    </paper>
+  </volume>
+  <volume id="taln" ingest-date="2024-06-27" type="proceedings">
+    <meta>
+      <booktitle>31ème Conférence sur le Traitement Automatique des Langues Naturelles, volume 1 : articles longs et prises de position</booktitle>
```

Comment:
For consistency with previous volume names, this should have a "Actes de la" prefix.

![image](https://github.com/acl-org/acl-anthology/assets/455256/b1d289fe-6ffd-4fa3-9340-a519c1860040)